components/perf-tools/msr-safe/SPECS/msr-safe.spec

```diff
@@ -0,0 +1,133 @@
+#---------------------------------------------------------------------------bh-
+# This RPM .spec file is part of the OpenHPC project.
+#
+# It may have been modified from the default version supplied by the underlying
+# release package (if available) in order to apply patches, perform customized
+# build/install configurations, and supply additional files to support
+# desired integration conventions.
+#
+#----------------------------------------------------------------------------eh-
+
+# msr-safe.spec
+
+%include %{_sourcedir}/OHPC_macros
+%{!?PROJ_DELIM: %global PROJ_DELIM -ohpc}
+
+# Base package name
+%define pname msr-safe
+%define PNAME MSR-SAFE
+
+
+Name:           msr-safe
```

Comment:
@crbaird The Name: should be:
msr-safe%{PROJ_DELIM}

This also needs to change in the geopm.spec Requires: field.

Reply:
Done. Will be in next pull request rev.